identityClient/api.go

```diff
@@ -0,0 +1,25 @@
+package identityClient
+
+// Sovereign represents the top level user of a realm
+// (i.e. a user who is able to log into the realm,
+// create/update/delete/view the realm properties and it's identities.)
+type Sovereign struct {
+	Id       int64  `json:"id"`       // Service defined unique identifier for the sovereign.
+	Name     string `json:"name"`     // User defined sovereign identifier.
+	Password string `json:"password"` // The sovereign's realm administration console login password.
```

Comment:
Does the Sovereign need a password if we can authenticate a signed request in the Keycloak instance?

Reply:
Good question. For now, yes, because the sovereign needs to be able to log into their realm/keycloak, which is what we're doing for mvp(@eddier) . Post launch though I want to write a keycloak extension/ authentication  SPI that will allow sovereigns to log into their realms/keycloak using their tozny account. Currently the sovereign is solely for interacting with the realm, while programatic access and control of Tozny Identity Management Service resources such as realms, sovereigns, consuming identities(consuming app users) is done with a (yet to be built) tozny sdk(s) identity client (which will use their existing tozny API credentials/keys) by talking to the identity service api which acts as the control plane/abstraction over the keycloak API.